README.md

```diff
@@ -17,43 +17,50 @@ comments.
 
 ## Repository Structure
 
-Each of the following top-level folders in this repo contain a DocSet that is published to
-[Microsoft Docs](https://docs.microsoft.com/powershell).
-
-- [/reference/](https://docs.microsoft.com/powershell/scripting/) is for PowerShell conceptual
-  topics and module reference across versions 5.1, 6.0, and 7.0. This content is also the source of
-  help content retrieved by the `Get-Help` cmdlet.
-  - [docs-conceptual/](https://docs.microsoft.com/powershell) - this folder contains the conceptual
-    documentation and the following docsets:
-    - [developer/](https://docs.microsoft.com/powershell/scripting/developer/) is the PowerShell SDK
-      documentation (migrated from MSDN)
-    - [dsc/](https://docs.microsoft.com/powershell/scripting/dsc/) is for the Desired State
-      Configuration feature
-    - [gallery/](https://docs.microsoft.com/powershell/scripting/gallery) is for the
-      [PowerShell Gallery](https://www.powershellgallery.com/)
-    - [jea/](https://docs.microsoft.com/powershell/scripting/learn/remoting/jea/overview) is for the Just Enough
-      Administration feature
-    - [wmf/](https://docs.microsoft.com/powershell/scripting/windows-powershell/wmf/overview) contains release notes
-      for the Windows Management Framework, the package used to distribute new versions of
-      PowerShell to previous versions of Windows.
+The following list describes the main folders in this repository.
+
+- `.github` - contains configuration settings used by GitHub for this repository
+- `.vscode` - contains configuration settings and recommended extensions for Visual Studio Code (VS
+  Code)
+- `assets` - contains downloadable files linked in the documentation
+- `reference` - contains the documentation published to
+  [docs.microsoft.com]([https://docs.microsoft.com/powershell/scripting/). This includes both
+  reference and conceptual content.
+  - `5.1` - contains the cmdlet reference and about topics for PowerShell 5.1
+  - `6` - contains the cmdlet reference and about topics for PowerShell 6
+  - `7.0` - contains the cmdlet reference and about topics for PowerShell 7.0
+  - `7.1` - contains the cmdlet reference and about topics for PowerShell 7.1
+  - `bread` - contains the TOC used for breadcrumb navigation
+  - `docs-conceptual` - contains the conceptual articles that are published to the Docs site. In
+    general, the folder structure mirrors the Table of Contents (TOC).
+  - `mapping` - contains the version mapping configuration used by the build system
+  - `media` - contains image files used in documentation. There are media folders throughout the
+    `conceptual` content. See the Contributor Guide for information on using images in documentation.
+  - `module` - contains the markdown source for the Module Browser page
+- `tests` - contains the Pester tests used by the build system
+- `tools` - contains other tools used by the build system
+
+> [!NOTE]
```

Comment:
I'd remove this line since it's OPS syntax and doesn't render as expected on GitHub.

Reply:
Fixed